fhir/immunizations_demo/docs/README.md

```diff
@@ -33,9 +33,25 @@ a project and configure the Cloud Healtcare API. However, instead of creating a
 service account, follow the instructions on the [OAuth
 quickstart](https://cloud.google.com/docs/authentication/end-user). You
 will only need the client ID, the client secret (API key) is not required for
-this use case. You will also need to register
+this use case.
+
+You will also need to register
 `http://localhost:4200/oauth-callback` as an authorized redirect URI on the
-OAuth configuration page. Change `localhost:4200` if you plan to develop on a
+OAuth configuration page. Follow these steps:
+
+1. If you haven't already, select **APIs & Services** from the left-hand menu of the Google Cloud Platform dashboard.
+
+2. Select **Credentials**. You should see a list of your registered projects under **Your OAuth 2.0 client IDs** (assuming you've created an OAuth client ID).
```

Comment:
Instead of assuming something that hasn't been included in the instructions so far, you can include a step for how to create a client ID.

Reply:
Sounds good! But isn't that kind of covered by this link? 

"However, instead of creating a
service account, follow the instructions on the [OAuth
quickstart](https://cloud.google.com/docs/authentication/end-user). You
will only need the client ID, the client secret (API key) is not required for
this use case."?

Maybe I should just get rid of the parenthetical. Let me know what you think.

If you want to add instructions for that extra step, how would you recommend organizing that portion of the README?

Reply:
Good point, I forgot I included that quickstart. Looks fine now.